chain/vm/invoker.go

```diff
@@ -173,17 +175,21 @@ func (*ActorRegistry) transform(instance invokee) (nativeCode, error) {
 				paramT := meth.Type().In(1).Elem()
 				param := reflect.New(paramT)
 
+				rt := in[0].Interface().(*Runtime)
 				inBytes := in[1].Interface().([]byte)
 				if err := DecodeParams(inBytes, param.Interface()); err != nil {
-					aerr := aerrors.Absorb(err, 1, "failed to decode parameters")
+					ec := exitcode.ErrSerialization
+					if rt.NetworkVersion() < network.Version7 {
+						ec = 1
+					}
```

Comment:
Although I don't think it's possible to hit, the encoding of params also uses this faulty exit code, perhaps do the same for this or make it fatal?

Reply:
@austinabell Thanks for the suggestion! If `SerializeParams` fails, it's treated as a fatal error in the one place it's called (`TryCreateAccountActor`), so the exitcode doesn't matter.

I'm changing its returned exitcode to `ErrSerialization` (since that's objectively correct), but I'm not putting it behind network upgrade logic.